### DISCUSSION TITLE
major upgrade 0.8.6 ; any specific  recommandation ?

### DISCUSSION THREAD

**Discussion by dcpc007** [felddy/foundryvtt-docker#207] [Q&A] [answered]
Hi, except backup all data, any specific opeation or recommandation before pull and up ?

**Comment by dcpc007** [comment]
Tip on my linux VPS for backuping with less time :

apt install lbzip2 (module for multi-thread compression mode)

cd /srv/foundryvtt
sudo tar -I lbzip2 -cvf foundryvtt-20210531.tar.bz2 Config Data Logs
--> this will use all the cores available at 100%, and not only one like a basic tar bz2 command.

**Comment by felddy** [comment, marked as answer]
If you've backed up your data you should be good-to-go!  Expect there to be some lag in module support for the `0.8.x` release.  

Good luck!

**Reply by dcpc007** [reply]
Ok that's the basics i've already done. Seems no other needs, thx !

To be sure, after my upgrade, are my old tar.bz2 backup usable ? or structure/format changed, and i must  create a new backup and delete the olders
